src_Core/CPU/EX_ALU_functions.bsv

```diff
@@ -1902,6 +1910,9 @@ function ALU_Outputs fv_CHERI (ALU_Inputs inputs, WordXL ddc_base);
         end
         alu_outputs.cap_val1 = result.value;
         alu_outputs.val1_cap_not_int = True;
+`ifdef PERFORMANCE_MONITORING
+        alu_outputs.set_offset_in_bounds = result.exact;
```

Comment:
This is not a good name for it. `result.exact` can be `True` even when outside the bounds of the capability.